server/routes/fehler.js

```diff
@@ -1,9 +1,42 @@
 const express = require("express");
 const router = express.Router();
-//const pool = require("./db");
+const pool = require("../db");      
 
 router.get("/", (req, res) => {
   res.send("Hello World");
 });
 
+//create Fail 
+router.post("/", async (req,res) => {   
+  try{
+    const{ titel, beschreibung, lösung, auswirkung, status, softwareid, anwenderid } = req.body;
+    const newFehler = await pool.query("INSERT INTO fehler (titel, beschreibung, lösung, auswirkung, status, softwareid, anwenderid) VALUES($1, $2, $3, $4, $5, $6, $7) RETURNING *",
+    [titel,beschreibung, lösung, auswirkung, status, softwareid, anwenderid ]); 
+
+    res.json ({
```

Comment:
Hier fehlt in der JSON rückgabe noch error: false

Hätten wir vorher nesse rkommunizieren müsssen damit wir das einheitlich machen zu Huschka